test/create.js

```diff
@@ -57,6 +57,35 @@ test('create() classic', (t) => {
   )
 })
 
+test('create() classic "too big"', (t) => {
+  const content = {
+    type: 'post',
+    text: Array(10000).fill('!').join('')
+  }
+  db.create(
+    { feedFormat: 'classic', content, },
+    (err, msg) => {
+      const comment = 'should error when content > 8kb'
+      if (!err) t.fail(comment)
+      else t.match(err.message, /content must be less than 8192/, comment)
+
+      const content = {
+        type: 'post',
+        text: Array(8192 - 114).fill('!').join(''),
+        recps: [sbot.id]
+      }
+
+      db.create({ feedFormat: 'classic', content }, err => {
+        const comment = 'should error when content > 8kb'
+        if (!err) t.fail(comment)
+        else t.match(err.message, /content must be less than 8192/, comment)
+
+        t.end()
+      })
+    }
+  )
+})
```

Comment:
This test needs to be moved to https://github.com/ssbc/ssb-classic/pull/2

Why? Because this test would improve test coverage (see our npm script "coverage") on ssb-classic, but in ssb-db2 it wouldn't affect test coverage.

Reply:
This test needs to stay - it was this test that surfaced that there was no error handling on validation. That's why there are try-catches now.

Actually why is there no explicit error handling? Seems odd to me